maintain-tidb-using-tiup.md

```diff
@@ -17,7 +17,7 @@ This document describes the following common operations when you operate and mai
 
 > **Note:**
 >
-> Since TiDB v4.0, PingCAP no longer provides support for TiDB Ansible. Since TiDB v5.0, PingCAP no longer provides TiDB Ansible documents. If you want to read the document that introduces how to maintain a TiDB cluster using TiDB Ansible, see [TiDB Ansible Common Operations (v4.0)](https://docs.pingcap.com/tidb/v4.0/maintain-tidb-using-ansible).
+> TiUP is the recommended method to maintain a TiDB cluster. Support for TiDB Ansible was deprecated in TiDB v4.0 and the TiDB Ansible documentation has been removed since TiDB v5.0.
```

Comment:
Because you've changed the redirects for Ansible documents for `dev` to go to v4.0, why is this note still needed at all?

It is still confusing, because it doesn't explain why this message is here. If I am a new user, what is Ansible? It would need some context such as "TiUP is the recommended upgrade path for Ansible users".

Reply:
Shall we remove the note?

Reply:
Yes, it looks better removed.